pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -1054,11 +1054,11 @@ def test_tdi_mul_float_series(self, box_df_fail):
         idx = tm.box_expected(idx, box)
 
         rng5f = np.arange(5, dtype='float64')
-        expected = TimedeltaIndex(rng5f * (rng5f + 0.1))
+        expected = TimedeltaIndex(rng5f * (rng5f + 1.0))
         box2 = pd.Series if box is pd.Index else box
         expected = tm.box_expected(expected, box2)
 
-        result = idx * Series(rng5f + 0.1)
```

Comment:
@jbrockmendel why did you change this? Left-over from initially changing the behaviour on floats?

Reply:
Yes.